packages/ngworker/lumberjack/http-driver/src/lib/configuration/lumberjack-http-driver.module.ts

```diff
@@ -44,7 +51,12 @@ export class LumberjackHttpDriverModule {
       providers: [
         {
           provide: lumberjackHttpDriverConfigToken,
-          useValue: options,
+          deps: [lumberjackLogDriverConfigToken],
+          useFactory: (logDriverConfig: LumberjackLogDriverConfig) => ({
```

Comment:
suggestion: add explicit return type to prevent type errors.
```suggestion
          useFactory: (logDriverConfig: LumberjackLogDriverConfig): LumberjackHttpDriverInternalConfig => ({
```